ckanext/switzerland/helpers.py

```diff
@@ -504,49 +501,13 @@ def _get_datetime_from_isoformat_string(dt_string):
         return False
 
 
-def convert_datetimes_for_display(dataset_or_resource_dict):
```

Comment:
This helper that converts Europe/Zurich datetimes to UTC is no longer needed, as dates will all be stored as UTC now.